contents/blog/secrets-of-posthog-query-performance.md

```diff
@@ -0,0 +1,174 @@
+---
+date: "2022-03-16" # :TODO
+title: "Secrets of PostHog query performance"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/launch-week-teaser.jpeg # :TODO:
```

Comment:
cc @lottiecoxon this needs changing. You mentioned under issue you have an image planned.

Reply:
It's ok, it's on Figma already I believe. I need to export, so it's an action for me.